stack-nodejs-16/stack.toml

```diff
@@ -0,0 +1,23 @@
+id = "io.buildpacks.stacks.ubi8"
+homepage = "https://github.com/paketo-community/ubi-base-stack"
+maintainer = "Red Hat"
```

Comment:
We might want to mark this as Paketo Community, what do you think? I'm not entirely sure what would be the right thing here @paketo-community/stacks-maintainers

Reply:
That was just what I had from earlier cut/paste would be happy to have it ias "Paketo Community"

Reply:
Pushed commit to make maintainer in all thre stack files say "Paketo Community"